sandbox/grommet-app/src/pages/layouts/kinds/Form/Form.tsx

```diff
@@ -44,7 +44,7 @@ export const Form = () => {
   const [numFiles, setNumFiles] = useState(0);
 
   return (
-    <Page pad={{ bottom: 'large' }} kind="narrow">
+    <Page pad={{ bottom: 'large' }} kind="narrow" flex={false}>
```

Comment:
I'm getting a horizontal scroll on this page which I wouldn't expect.

<img width="1726" alt="Screenshot 2025-02-20 at 10 58 34 AM" src="https://github.com/user-attachments/assets/af0bedcb-3d5b-4f35-9db0-73c06f4b7eb9" />